### ISSUE TITLE
Doesn't works well with ScrollView

### ISSUE THREAD

**Issue by vyndor** [worker8/TourGuide#90]
I have troubles when play a TourGuide on a View inside a ScrollView. I think overlay should scroll with the content.

**Comment by 10K-with-AI-Enterprise-Edition** [comment]
Same here. I have this issue too.
I think the developer is not supporting this library anymore?

**Comment by dduvedi** [comment]
any luck on this?

**Comment by 10K-with-AI-Enterprise-Edition** [comment]
Yep, I gave up on this library. I'm using a library called simple-tooltip.

https://github.com/douglasjunior/android-simple-tooltip
